src/schema/v2/artworksForUser/artworksForUser.ts

```diff
@@ -49,16 +49,20 @@ export const artworksForUser: GraphQLFieldConfig<void, ResolverContext> = {
       context
     )
 
-    const remainingSize = (gravityArgs.size || 0) - newForYouArtworks.length
     const backfillArtworks = await getBackfillArtworks(
-      remainingSize,
+      size || 0,
       args.includeBackfill,
       context,
       args.onlyAtAuction
     )
 
-    const artworks = [...newForYouArtworks, ...backfillArtworks]
-
+    const artworks = newForYouArtworks
+      .concat(
+        backfillArtworks.filter(
+          (item2) => !newForYouArtworks.some((item1) => item1.id === item2.id)
+        )
+      )
+      .slice(0, size)
```

Comment:
This eliminates repeat entries, by `id`, then trims to the requested size.

That `id` property was available in my local tests, but is it safe to assume? Is there a more idiomatic way to accomplish this?

Reply:
The underlying data (`newForYouArtworks` and `backfillArtworks`) can be traced back, but they're the result of data loaders that hit Gravity API endpoints which return a list of artworks (via their `short` JSON).

So, the objects here, and their `id` property - map to that data shape. Indeed, `id` (maybe even better `_id`) is available here in reality (not just in tests), and looks like a good attribute to use for the uniqueness filter.

Reply:
Could also do 
```
const artworks = uniqBy(foo.concat(bar), 'id').slice(0.size)
```